TrafficCapture/README.md

```diff
@@ -60,8 +60,8 @@ Partial example output of the OpenSearch Benchmark tool:
 Running opensearch-benchmark w/ 'geonames' workload...
 
    ____                  _____                      __       ____                  __                         __
-/ __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
-/ / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
+  / __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
+ / / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
```

Comment:
Revert these lines?

Reply:
This corrects the output from OSB. It uses an italicized font and requires those spaces to make the big characters legible